src/GraphClient.ts

```diff
@@ -79,4 +81,32 @@ export default class GraphClient {
     // Creates and returns graph client
     return new GraphClient(apolloClient);
   }
+
+  /**
+   * This method adds callback to subscription client. Currently, it logs the
+   * different callbacks. In future these callbacks, can be useful to design error
+   * handling and retry mechanisms.
+   *
+   * @param subscriptionClient Instance of subscription client.
+   */
+  private static attachSubscriptionClientCallbacks(subscriptionClient: SubscriptionClient) {
+    subscriptionClient.onConnected(() => {
+      Logger.info('Connected to the graph node');
+    });
+    subscriptionClient.onReconnected(() => {
+      Logger.info('Reconnected to the graph node');
+    });
+    subscriptionClient.onConnecting(() => {
+      Logger.info('Connecting to the graph node');
+    });
+    subscriptionClient.onReconnecting(() => {
+      Logger.info('Reconnecting to the graph node');
+    });
+    subscriptionClient.onDisconnected(() => {
+      Logger.info('Disconnected to the graph node');
+    });
+    subscriptionClient.onError((error) => {
+      Logger.error(`Error connecting to graph node. Reason: ${error.message}`);
```

Comment:
> We might want to think of shutting down the process if the connection couldn't be established after the max number of retries we could make.
> Thoughts ?

We can raise SIGTERM in case of error.  Facilitator will need to stop in this case because max retries has a max limit. We will handle the SIGTERM in the ticket #19